### PR TITLE
add note about new licenses

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3326#event-2941860662

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: Teams/room-systems/with-exchange-online.md [hunk 1]
```diff
@@ -140,6 +140,9 @@ Import-PSSession $Session -DisableNameChecking
 6. Click **Save**.
 
 For validation, you should be able to use any Skype for Business client to log in to this account.
+
+> [!NOTE]
+> If you are currently using E1, E3, E4, E5 SKUs with Skype for Business Plan 2 with Audio Conferencing or with Office 365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model described [here](https://docs.microsoft.com/en-us/MicrosoftTeams/room-systems/skype-room-systems-v2), after current licenses expire.
```
Comment (JohanFreelancer9) [inline Teams/room-systems/with-exchange-online.md]:
```suggestion
> If you are currently using E1, E3, E4, E5 SKUs with Skype for Business Plan 2 with Audio Conferencing or with Office 365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model described in [Teams Meeting Room Licensing Update](https://docs.microsoft.com/MicrosoftTeams/room-systems/skype-room-systems-v2), after current licenses expire.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/room-systems/with-exchange-online.md (hunk 1 above)
Comment (LanaChin) [reply Teams/room-systems/with-exchange-online.md]: @VLG17 - can you commit these edits? I don't have the permissions to do that. Thanks!

Comment (VLG17) [reply Teams/room-systems/with-exchange-online.md]: @LanaChin done!

**Comment by LanaChin** [comment]
@VLG17, @scanum - The updates are now live, https://docs.microsoft.com/MicrosoftTeams/rooms/with-exchange-online.  The folder structure of this file has changed so I opened another PR and made the changes using that PR. Am closing this PR without merging it. Thanks.

**Pull request closed without merging**